web-console/developer/pipelines/pipelines.md

```diff
@@ -1,5 +1,314 @@
 # Pipelines
-The Pipelines page is a primary nav item in the OpenShift Developer perspective.
+The Pipelines page is a primary navigation item in the OpenShift Developer perspective.  Pipelines is only shown when the operator is installed.
+
+## Pipelines Resource Table
+![Pipelines Resource Table](img/PL-ListView.png)
```

Comment:
I saw the Tech Preview stuff come through previously and I think the designs look great. It hit me looking at this that the Tech Preview label looks like a button. Maybe it's worth considering a badge look and feel? Ignore me completely if you already considered a million options and you settled on this anyway ;)

Reply:
Noticing now that this one has more of a drop shadow...the images following below look a bit different and less like buttons.

Reply:
I'll update that one mock - thanks for catching, i missed it!